modules/nw-ingress-gateway-api-troubleshooting-degraded.adoc

```diff
@@ -0,0 +1,59 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-gateway-api-troubleshooting-degraded_{context}"]
+= Ingress Operator is Degraded due to Gateway API and OSSM conflict
+
+In [product-title] 4.20 and later, if you create a `GatewayClass` resource while a conflicting OpenShift Service Mesh (OSSM) v2.x subscription exists, the `ingress` Cluster Operator (CIO) reports a `Degraded` status. This procedure details how to verify and resolve this conflict.
+
+This occurs because the Gateway API implementation requires OSSM v3.x, which cannot coexist with OSSM v2.x. The CIO detects this conflict, stops the Gateway API provisioning, and reports the `Degraded` status to alert administrators.
+
+.Verification
+
+. Check the `ingress` Cluster Operator status:
++
+[source,terminal]
+----
+$ oc get clusteroperator ingress -o yaml
+----
+
+. In the `status` section of the output, look for a `Degaded` condition with `status: "True"` and a `reason: GatewayAPIOSSMConflict`.
++
+.Example Degraded status message
+[source,yaml]
+----
+status:
+  conditions:
+
+    lastTransitionTime: "2025-10-22T17:00:00Z"
+
+    message: 'Failed to install OpenShift Service Mesh 3.x for Gateway API: A
+      conflicting OpenShift Service Mesh 2.x subscription was found. Remove the
+      GatewayClass resource or the conflicting OSSM 2.x subscription to resolve.'
+    reason: GatewayAPIOSSMConflict
+    status: "True"
+    type: Degraded
+----
+
+.Procedure
+
+To resolve this issue and clear the `Degraded` status, choose one of the following options:
```

Comment:
Vale DITA might flag this floating line. Consider building this sentence into step 1 of the procedure.